eng/pipelines/dotnet-framework-wcf.yml

```diff
@@ -0,0 +1,12 @@
+trigger: none
+pr: none
```

Comment:
I think this should have a pr trigger on master.  e.g. https://github.com/microsoft/dotnet-framework-docker/blob/master/eng/pipelines/dotnet-framework.yml#L2

Reply:
Can/should path filters be specified to only trigger PR validation when WCF artifacts are changes?

Reply:
done

Reply:
I'm not sure we should use path triggers for PRs.  If we make changes to the pipeline infrastructure, I would want this pipeline to be validated for those changes.

Reply:
Multiple trigger paths are a possibility... The primary reason I mention this is that if a WCF only PR is made, I wouldn't want it to wait for the long running SDK tests to complete in order for it to turn green.  This type of scenario could waste a lot of compute.

That being said, I am fine with "tuning" these scenarios after the initial onboarding is complete.